docs/pom.xml

```diff
@@ -2786,7 +2786,7 @@
                         <quarkus-version>${project.version}</quarkus-version>
                         <surefire-version>${version.surefire.plugin}</surefire-version>
                         <graalvm-version>${graal-sdk.version-for-documentation}</graalvm-version>
-                        <graalvm-flavor>${graal-sdk.version-for-documentation}-java11</graalvm-flavor>
+                        <graalvm-flavor>${graal-sdk.version-for-documentation}-java17</graalvm-flavor>
```

Comment:
I know if we want to set all documentation reference to java 17 as quarkus is still based on Java 11

Reply:
Reverted.